File: ejercicios/python/ejercicio087.py
from random import randint

def llenado_aleatorio(m):
    for i in range(3):
        m.append([randint(0, 100) for i in range(4)])

def mostrar_matriz(m):
    for i in m:
        for j in i:
            print(j, end='\t')
        print("")
        
if __name__ == "__main__":
    matriz=[]
    llenado_aleatorio(matriz)
    print("matriz original")
    mostrar_matriz(matriz)
    
    divisor = matriz[2][2]
    for i in range(3):
        for j in range(4):
            matriz[i][j] = int(matriz[i][j] / divisor)
    
    print("matriz dividida")
    mostrar_matriz(matriz)
    

File: ejercicios/c++/ejercicio087.cpp
#include <iostream>
#include <cstdlib>
#include <ctime>
using namespace std;
//procedimiento que llena aleatriamente la matriz
void llenado_aleatorio(int[3][4]);

//procedimiento que muestra en pantalla la matriz
void ver_matriz(int[3][4]);

int main(int argc, char *argv[]) {
	int matriz[3][4];
	int divisor;
	
	srand(time(0));
	llenado_aleatorio(matriz);
	
	cout << "matriz original" << endl;
	ver_matriz(matriz);
	
	divisor = matriz[2][2];
	for(int f=0; f<3; f++){
		for(int c=0; c<4; c++){
			matriz[f][c]/=divisor;
		}
	}
	
	cout << "matriz dividida" << endl;
	ver_matriz(matriz);
	
	return 0;
}

void llenado_aleatorio(int m[3][4]){
	for(int f=0; f<3; f++){
		for(int c=0; c<4; c++){
			m[f][c] =1 + rand() % 9;
		}
		cout << endl;
	}
}
void ver_matriz(int m[3][4]){
	for(int f=0; f<3; f++){
		for(int c=0; c<4; c++){
			cout << m[f][c] << "\t";
		}
		cout << endl;
	}
}

